### ISSUE TITLE
Beanshell script for ST_StrahlerStreamOrder

### ISSUE THREAD

**Issue by agouge** [orbisgis/gdms-topology#32]
Write a beanshell script / tutorial on how to use the new ST_StrahlerStreamOrder function and post it to the wiki.

**Comment by agouge** [comment]
The tutorial can be found [here](https://github.com/agouge/gdms-topology/wiki/Strahler-stream-order). It will be merged into the wiki once PR #25 is merged.
